### PR TITLE
docs: update docs

### LINKED ISSUE #303: docs: update to make it easier understandable
- [ ] add finetuning to the docs
- [ ] bff description
- [ ] production readiness (QPS 13, latency 300 ms)    
- [ ] security
- [ ] s3 bucket
- [ ] also add google analytics
- [ ] find out more things to add

### PR DESCRIPTION
**Pull request by azayz**
closes #303

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,66 @@ jina now stop
 - 10 GB assigned to docker
 - User must be permitted to run docker containers
 
+### Production Readiness
```
Comment (ZiniuYu) [inline README.md:138]: Make sure to stick to one title capitalization rule

**Review by makram93 — COMMENTED** [review]
- I think this is too much information to be added in a single page. Can you divide it into subpages with appropriate headings? You can take Jina or DocArray docs as a reference. It does not matter if the content is less for a page... at least the user will have a proper overview instead of scrolling a single page for all information.
- Could you also update the gif - it is based on the old version. If it is too much/difficult work then maybe we can pull [this section](https://jina.ai/#:~:text=to%20create%20value.-,Neural%20Search,Deep%20learning%2Dpowered%20information%20retrieval%20on%20cross/multi%2Dmodal%20data.,-Text%2Dto%2Dimage) from jina website and add it here as well

File: docs/user-guides/production.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Production readiness
+
+Jina NOW apps are production ready, our goal is to provide you with a search pipeline that is easy to integrate in your production
+system. We also come with other aspects such as AWS S3 Bucket data integration, security and finetuning.
+From our experiments you can expect:
```
Comment (ZiniuYu) [inline docs/user-guides/production.md:5]: Can we provide more details like the app we play with, the dataset we use, and the hardware config?

File: docs/user-guides/production.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+<img width="506" alt="question-emails" src="https://user-images.githubusercontent.com/40893766/191207810-e1a1df17-acf3-4de2-8211-9a297b52e3a8.png
+">
+
+## How to update your API keys
+
+To update your API keys use this simple script:
+
+```bash
+python now/admin/update_api_keys.py
```
Comment (ZiniuYu) [inline docs/user-guides/production.md:58]: Where is this `update_api_keys` file? IMO this operation should better be included in CLI

File: docs/user-guides/production.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Finetuning
+
+We also support finetuning, to enable this when you create your `DocumentArray` you also provide labels inside
+the tags more specifically the tag `finetuner_label`. This will automatically trigger the finetuning and your embeddings
+will be improved thus enhancing the quality of your final NOW search application.
+
+More information on the training data format can be found [here](https://finetuner.jina.ai/walkthrough/create-training-data/#prepare-training-data)
+
+## Backend for frontend
```
Comment (joschkabraun) [inline docs/user-guides/production.md:61]: Missing capitalization in frontend

**Review by joschkabraun — APPROVED** [review]
Nice update on the docs, I really like it!

**Review by ZiniuYu — APPROVED** [review]

**Comment by ZiniuYu** [comment]
well done 🙌
